config/agencies.yml

```diff
@@ -48,7 +48,7 @@ development:
 
 production:
   1:
-    name: 'CBP'
+    name: 'DHS'
```

Comment:
Will this change the name in the database or is this just updating this to be in line with our usage elsewhere?

Reply:
It will update the db.  I don't believe it's currently used anywhere (a check for CBP or DHS)